v2.0/monitor-cockroachdb-with-prometheus.md

```diff
@@ -39,10 +40,15 @@ CockroachDB generates detailed time series metrics for each node in a cluster. T
 
 1. Download the starter [Prometheus configuration file](https://github.com/cockroachdb/cockroach/blob/master/monitoring/prometheus.yml) and [aggregation rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/aggregation.rules) for CockroachDB:
```

Comment:
The `aggregation.rules` file is now called `aggregation.rules.yml`. See https://github.com/cockroachdb/cockroach/pull/24253/files